ee/allowedcmd/cmd.go

```diff
@@ -12,6 +12,7 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"errors"
```

Comment:
It looks like the lint job is complaining because this import is not added alphabetically -- same goes for other file as well